bqemulatormanager/manager.py

```diff
@@ -69,7 +69,7 @@ def load(self, data, path):
 
     def create_dataset(self, dataset_name: str, exists_ok=True):
         dataset = bigquery.Dataset(f'{self.project_name}.{dataset_name}')
-        self.client.create_dataset(dataset)
+        self.client.create_dataset(dataset, exists_ok=exists_ok)
```

Comment:
This change looks OK, and I also think we should check this kind of bug automatically by using a linter.

Maybe this repo? https://github.com/nhoad/flake8-unused-arguments

Reply:
I added. 5c6b0ea

Reply:
Thanks!